components/config/src/config/mod.rs

```diff
@@ -98,6 +98,10 @@ pub struct Config {
     pub markdown: markup::Markdown,
     /// All user params set in `[extra]` in the config
     pub extra: HashMap<String, Toml>,
+    /// Disable the generation of Sitemap.xml
+    pub no_sitemap: bool,
+    /// Disable the generation of robots.txt
+    pub no_robots: bool,
```

Comment:
Let's make those:

```
generate_sitemap
generate_robots_txt
```
with a default of `true`

Reply:
Implemented (see https://github.com/getzola/zola/pull/2579/commits/63ef902ad8f5ecfc849137f24a98dddf5bed055a).